test/stability/testCksum/testCksum.go

```diff
@@ -47,7 +47,7 @@ const (
 )
 
 var (
-	totalPackets uint64 = 10
+	totalPackets uint64 = 32
```

Comment:
Was this tested both at physical machines and virtual machines?

Reply:
yes. 32 is more logical cause its the burst size.